app/controllers/users_controller.rb

```diff
@@ -0,0 +1,15 @@
+class UsersController < ApplicationController
+  # before_action :set_user, only: %i[show update destroy]
```

Comment:
- [ ] Don't forget to either uncomment this or remove this line if you aren't using it.